cdn/rpcserver/rpcserver.go

```diff
@@ -95,12 +95,13 @@ func (css *Server) ObtainSeeds(ctx context.Context, req *cdnsystem.SeedRequest,
 			PeerId:   peerID,
 			HostUuid: hostID,
 			PieceInfo: &base.PieceInfo{
-				PieceNum:    int32(piece.PieceNum),
-				RangeStart:  piece.PieceRange.StartIndex,
-				RangeSize:   piece.PieceLen,
-				PieceMd5:    piece.PieceMd5,
-				PieceOffset: piece.OriginRange.StartIndex,
-				PieceStyle:  piece.PieceStyle,
+				PieceNum:     int32(piece.PieceNum),
+				RangeStart:   piece.PieceRange.StartIndex,
+				RangeSize:    piece.PieceLen,
+				PieceMd5:     piece.PieceMd5,
+				PieceOffset:  piece.OriginRange.StartIndex,
+				PieceStyle:   piece.PieceStyle,
+				DownloadCost: piece.DownloadCost,
```

Comment:
`PieceInfo` needs to be consistent with the `PieceResult` of  `scheduler.proto`.

https://github.com/dragonflyoss/Dragonfly2/blob/main/pkg/rpc/scheduler/scheduler.proto#L96
https://github.com/dragonflyoss/Dragonfly2/blob/main/pkg/rpc/scheduler/scheduler.proto#L98